### PR TITLE
Improve some role descriptions

### PR DESCRIPTION
**Pull request by KingPanda360**

### CODE REVIEW

File: Resources/Lang/en_US.json [hunk 1]
```diff
@@ -887,19 +887,19 @@
...
   "PlagueBearerInfoLong": "(Apocalypse):\nAs the Plaguebearer, plague everyone using your kill button to turn into Pestilence.\nOnce you turn into Pestilence, you will become immortal and gain the ability to kill, and you will kill anyone who tries to kill you.\n\nAlso, when infected players interact with uninfected players, they will also be infected.",
-  "PestilenceInfoLong": "(Apocalypse):\nAs Pestilence, you're an unstoppable machine.\nAny attack towards you will be reflected towards them.\nIndirect kills don't even kill you.\n\nOnly way to kill Pestilence is by vote, or by it misguessing.\n<b>Your presence is announced to everyone at the meeting after you transform.</b>",
+  "PestilenceInfoLong": "(Apocalypse):\nAs Pestilence, you're an unstoppable machine.\nAny attack towards you will be reflected towards them.\nIndirect kills don't even kill you.\n\nOnly way to kill Pestilence is by vote or by misguessing.\n<b>Your presence is announced to everyone at the meeting after you transform.</b>",
```
Comment (0xDrMoe) [inline Resources/Lang/en_US.json:890]: This indicates the non-Pestilence player may misguess to exile Pestilence. Either reword it or re-add "it"

File: Resources/Lang/en_US.json [hunk 2]
```diff
@@ -887,19 +887,19 @@
...
+  "BakerInfoLong": "(Apocalypse): \nAs the Baker, you can use your kill button on a player per round to give them bread. \nOnce a set amount of players are alive with bread, you become Famine.\n\nIf the Bread gives additional effects and the setting is on, then you can vent to change the bread that you give out. \n<b>Bread Effects:</b>\nReveal: Reveals the target's role to the Baker (stays the whole game)\nRoleblock: Sets the target's kill cooldown to 999 (resets to normal after the meeting)\nBarrier: Gives the target a barrier that is only known to the Baker (barrier is removed after the meeting)",
+  "FamineInfoLong": "(Apocalypse): \nOnce the Baker has the set amount of people with bread alive, they will become Famine. Suppose Famine is not voted out after the meeting. Then they become Famine, and every player without bread will starve (excluding other Apocalypse members).\nAfter this starvation of everyone without bread, Famine can use their kill button to starve any remaining players, which will kill those players right before the next meeting.\n\n<b>You are invincible, and your presence is announced to everyone at the meeting after you transform.</b>",
```
Comment (0xDrMoe) [inline Resources/Lang/en_US.json:894]: This seems wrong to the tongue. Why are you saying "Suppose"? You can reword this significantly better.

File: Resources/Lang/en_US.json [hunk 3]
```diff
@@ -1005,7 +1005,7 @@
   "TrickyInfoLong": "(Add-ons):\nAs the Tricky, your kills will have a random death reason.",
   "TiredInfoLong": "(Add-ons):\nWhenever Tired kills (or uses kill ability on) someone, alternatively whenever they finish a task, they will temporarily get lower vision & lower speed.",
   "StatueInfoLong": "(Add-ons):\nWhenever many people are near the Statue, the Statue is completely frozen or slowed down depending on the settings.",
-  "EvaderInfoLong": "(Add-ons):\nWhen the Evader gets voted out, there is a chance the Evader will not get ejected. (Chance set by host.)",
+  "EvaderInfoLong": "(Add-ons):\nWhen the Evader gets voted out, there is a chance the Evader will not get ejected. (Chance set by the Host.)",
```
Comment (0xDrMoe) [inline Resources/Lang/en_US.json:1008]: Good, just remove the second "Evader in the sentence as it is redundant. Consider using "they".

File: Resources/Lang/en_US.json [hunk 4]
```diff
@@ -1021,7 +1021,6 @@
   "SlothInfoLong": "(Add-ons):\nThe Sloth's default movement speed is slower than others.\n(Speed depends on the setting of the Host)",
   "ProhibitedInfoLong": "(Add-ons):\nAs the Prohibited, you have specific vents that you can't use.\nHow many vents are disabled depends on the Host's settings.",
   "EavesdropperInfoLong": "(Add-ons):\nAs the Eavesdropper, you have a chance to read other role/addon information-based messages like Mortician or Sleuth.",
-
```
Comment (0xDrMoe) [inline Resources/Lang/en_US.json:1024]: Please re-add this line. It is the divider between addon descriptions and settings below.

**Review by 0xDrMoe — CHANGES_REQUESTED** [review]
Please amend the changes so approval can be given. Once amended, changes will be merged.

**Comment by Tommy-XL** [comment]
@KingPanda360 can you fix?
I want sync translations

**Comment by KingPanda360** [comment]
> @KingPanda360 can you fix? I want sync translations

Fixed

**Pull request merged**